src/apps/inventory/views.py

```diff
@@ -34,35 +34,52 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class ProductFilter(FilterSet):
```

Comment:
I don't understand the reason we need stock filter. Lets filter by in_stock - boolean filter? What do you think?

Reply:
Fair